test/test_suite.sh

```diff
@@ -48,6 +48,10 @@ export WDQS_PROXY_IMAGE_NAME="$WDQS_PROXY_IMAGE_NAME:latest"
 export QUICKSTATEMENTS_IMAGE_NAME="$QUICKSTATEMENTS_IMAGE_NAME:latest"
 export ELASTICSEARCH_IMAGE_NAME="$ELASTICSEARCH_IMAGE_NAME:latest"
 
-export DEFAULT_SUITE_CONFIG="--env-file default.env -f docker-compose.yml"
+# Begin with env vars in `template.env` in the example directory
+# to base our tests runs against our recommended default configuration.
+# Then add local testing `default.env`` which overrides selectively, as a user does
+# in the course of configuring a new Wikibase install.
+export DEFAULT_SUITE_CONFIG="--env-file ../example/template.env --env-file default.env -f docker-compose.yml"
```

Comment:
I'd really vote for clearer env file names, such as *base* and *override* or similar

Reply:
A for `test`, the notes above. For  `example` I think `template.env` would be better renamed to `default.env` (which I think we should considering including in the docker-compose file there) and then continue to rely upon the somewhat standard practice of using `.env` as the local overrides.